project/plugins.sbt

```diff
@@ -10,6 +10,10 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
 
 addSbtPlugin("com.thesamet" % "sbt-protoc" % "1.0.6")
 
+addSbtPlugin("com.codecommit" % "sbt-github-packages" % "0.5.3")
+
+addSbtPlugin("io.chrisdavenport" % "sbt-no-publish" % "0.1.0")
```

Comment:
@TonioGela, these plugins probably are not necessary. 
In case I remove them.

Reply:
How can they be optional sorry?

Reply:
Solved, it's a merge mess :D